src/models/chan.js

```diff
@@ -15,6 +15,11 @@ Chan.Type = {
 	SPECIAL: "special",
 };
 
+Chan.State = {
```

Comment:
This can be expanded for queries (#1700) later on to state if user is online or not.